### PR TITLE
Adjusting the small-caps font details in Witch-Hunt

### PR DESCRIPTION
**Pull request by Singulariity**
The title explains. I will probably complete them all soon.

### CODE REVIEW

File: story/ep3/wh/umi3_5.txt [hunk 1]
```diff
@@ -517,7 +517,7 @@
 `When the Stake nee-chan let out a thunderous cry, Ronove laughed frivolously and disappeared, as if running away... `
 `"...Hmph. `
 `Why am I, the oldest sister of the Seven Stakes of Purgatory, being summoned for menial tasks like this?! `
-`...Even though Beato-sama could've told one of the younger sisters, who look like they've got nothing better to do. Sheesh...!" `
+`...Even though {f:5:Beatrice}-sama could've told one of the younger sisters, who look like they've got nothing better to do. Sheesh...!" `
```
Comment (Singulariity) [inline story/ep3/wh/umi3_5.txt:520]: Lucifer doesn't refer Beatrice as Beato. Fixed.

File: story/ep4/wh/umi4_17.txt [hunk 2]
```diff
@@ -708,7 +708,7 @@
 `At a glance, it felt as though she still hated me for something I'd done to her...and that she blamed me for it. `
 `However, ...I naturally didn't have a clue, and Beato had said it herself. `
 `Six years ago, there was no link between us. `
-`Six years ago, no person called Beatrice existed for me. `
+`{p:1:Six years ago, no person called {f:5:Beatrice} existed for me}. `
```
Comment (Singulariity) [inline story/ep4/wh/umi4_17.txt:711]: This text should be red

**Comment by vit9696** [comment]
Looks good to me, merged)

**Comment by vit9696** [comment]
As for the links, need to update version and changelog.

**Pull request merged**